sky/tools/create_macos_framework.py

```diff
@@ -114,6 +115,14 @@ def process_framework(dst, args, fat_framework, fat_framework_binary):
 
     subprocess.check_call(["strip", "-x", "-S", fat_framework_binary])
 
+  # Zip FlutterMacOS.framework.
+  if args.zip:
+    subprocess.check_call([
+        'zip', '-r', '-y', 'FlutterMacOS.framework.zip',
```

Comment:
So `-y` worked and the symlinks are maintained?

Reply:
For recipes v2 we'll need a follow up or a work around. The Zip command is correct the problem is in the CAS service that follows symlinks by default. I'm taking a look at the service to see if I can implement some logic to preserve symlinks and if it is too complicated I'll send a follow up to regenerate the symlinks in this script.